3600/3/helpers/comment_helper.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module CommentsHelper
+  def add_new_comment
+    @user = User.find(session[:user_id])
+    @comment = @user.comments.create(grade: params[:grade].to_i, text: params[:text])
+    @comment.restaurant_id = @restaurant.id
+    @comment.save
```

Comment:
Take a look at the source code of `create` method https://apidock.com/rails/ActiveRecord/Base/create/class